stream-applications-build/pom.xml

```diff
@@ -40,7 +40,7 @@
 		<nohttp-checkstyle.version>0.0.2.RELEASE</nohttp-checkstyle.version>
 		<disable.nohttp.checks>true</disable.nohttp.checks>
 		<spring-javaformat-checkstyle.version>0.0.7</spring-javaformat-checkstyle.version>
-		<spring-boot.version>2.6.7</spring-boot.version>
+		<spring-boot.version>2.7.0</spring-boot.version>
```

Comment:
What our version is this for?
Are we really planning a new minor version?
I mean it looks like there are some breaking changes pulled by this Spring Boot upgrade...

Well, it might not be a big deal for out-of-the-box apps, but if our functions are used in the end-user code they need to understand consequences.

Reply:
This is for the last release of `2021.1.x` line so the thinking was that we would get on the latest (non 3.x) Spring Boot and Spring Cloud (2021.0.3). 

Good point on the breaking property changes. Let's chat more at apps discussion today.